.github/workflows/core.yml

```
@@ -362,10 +362,10 @@ jobs:
           model: ${{ matrix.model }}
           lang: ${{ matrix.lang }}
           status: ${{ job.status }}
-        if: ${{ always() && env.ACTIONS_DO_UI_TEST == 'true' && !matrix.n4w1 }}
+        if: ${{ always() && env.ACTIONS_DO_UI_TEST == 'true' && !matrix.n1w1 }}
         continue-on-error: true
       - uses: ./.github/actions/upload-coverage
```

actionlint (1.7.12)

[error] description is required in metadata of "Upload coverage data" action at "/home/jailuser/git/.github/actions/upload-coverage/action.yml"

(action)